sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.cs

```diff
@@ -342,9 +343,69 @@ public async Task StartProcessingStartsTheProcessing(bool async)
             // Shut the processor down to ensure resource clean-up, but ignore any errors since it isn't the
             // subject of this test.
 
-            try
-            { await mockProcessor.Object.StopProcessingAsync(cancellationSource.Token); }
-            catch { }
+            try { await mockProcessor.Object.StopProcessingAsync(cancellationSource.Token); }  catch {}
```

Comment:
Not sure if intentional, but single-line try/catch blocks like this in this file have two blank spaces before the `catch` keyword. Just mentioning in case the extra space was an accident.

Reply:
It's intentional; I like the extra space when doing this kind of "ignore all the things" just to give some visual distance to the catch block.   I should consider doing an extension for `Task` in the next round for something like `SafeAwait` or a helper method or something.    I'll give it some thought and defer to the next round.